bin/start-libbitcoind.js

```diff
@@ -26,3 +26,32 @@ daemon.on('error', function(err) {
 daemon.on('open', function(status) {
   log.info('status="%s"', status);
 });
+
+function exitHandler(options, err) {
```

Comment:
It makes sense to have this in the bin scripts and not lib/ because other people using bitcore node in their own application may want to handle process signals differently. I'm up for having a util class in bin/ which can have the exit handler if we don't want to have the same code twice, or we can wait for your command line interface changes and deal with it then.